doc/da65.sgml

```diff
@@ -542,6 +542,18 @@ code. The following attributes are recognized:
   range, where <tt/label/ is the label name given with the <tt/NAME/
   attribute, and <tt/offs/ is the offset within the data.
 
+  <tag><tt>VOPERAND</tt></tag>
+  This optional attribute is followed by a numerical value. It tells the
+  assembler that subroutine calls to this label follow "virtual operands"
+  of the given bytes like this:
+
+<tscreen><verb>
+       	JSR	LabelWith2BytesOfVoperand
+	.byte	$00, $10    ; virtual operands
+	; return here
+	BIT	$0F
```

Comment:
BIT instructions are sometimes used to skip parts of instructions (see http://www.6502.org/tutorials/6502opcodes.html#BIT). Therefore I suggest that your sample code uses something else to avoid a misleading association for the reader.

Reply:
Changed the sample not to be misinterpreted as real codes.